vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -348,7 +350,7 @@ def set_relationship_on_hosts(hosts)
       #
 
       def normalize_blank_property_num(property)
-        property.try(:to_i)
+        property.try(:to_i) || 0
```

Comment:
hm, I see that the normalize methods are used only in scvmm code. I would swear it was on more places. 

I guess we should be filling issue for this, either for doing proper normalizing with logging error/warning, but not failing refresh. Or just deleting normalizing and throwing error when the input changes.